packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -769,3 +769,78 @@ class ImportedCluster extends Resource implements ICluster {
     }
   }
 }
+
+/**
+ * Properties for EksOptimizedAmi
+ */
+export interface EksOptimizedAmiProps {
+  /**
+   * What instance type to retrieve the image for (normal or GPU-optimized)
+   *
+   * @default Normal
+   */
+  readonly nodeType?: NodeType;
+
+  /**
+   * The Kubernetes version to use
+   *
+   * @default The latest version
```

Comment:
Shouldn’t this match the cluster version?

Use “-“ as the default value if the value is not specific

Reply:
It would match the cluster version, but if it's not provided to the Props it'll default to the latest. That makes sense to me, but I can elaborate the description to include it.

Reply:
I've added the `-`